tests/contrib/subjects/conftest.py

```diff
@@ -21,9 +22,33 @@
 def subject_full_data():
     """Controlled vocabulary backed subject data."""
     return {
-        "id": "https://id.nlm.nih.gov/mesh/D000001",
-        "scheme": "MeSH",
-        "subject": "Calcimycin",
+        "subject": {
+            "en": "Dark Web",
+            "de": "Darknet",
+            "fr": "Réseaux anonymes (informatique)",
+        },
+        "id": "1062531973",
```

Comment:
`id` has to be a URI. Subjects are not "scheme-spaced" they are all in the same container so they have to be distinct from any other future potential subject id.

Reply:
I understand your point and wasn't aware of this. So far we had some subjects (like DDC) where no URI based id was available. This has to be clarified in the docs because this is very important for the implementation of export formats (e.g. for use with OAI-PMH -> most format largely depends on id's).